R/distance.R

```diff
@@ -62,37 +68,59 @@
 #'   return(as.dist(res))
 #' }
 #'
-#' seqDist(immdata$data[1:2], .method = f) # the result based on our custom distance defined above
+#' seqDist(immdata$data[1:2], .method = f, .group_by_seqLength = FALSE)
 #' @export seqDist
 
-seqDist <- function(.data, .col = "CDR3.nt", .method = "hamming", ...) {
+seqDist <- function(.data, .col = "CDR3.nt", .method = "hamming", .group_by = c("V.name", "J.name"), .group_by_seqLength = TRUE, ...) {
   .validate_repertoires_data(.data)
   sample_truth <- .data[[1]]
+  gb_absent <- all(is.na(.group_by))
```

Comment:
Modify the variable name or leave a comment please. It's challenging to infer the meaning by its name